app/src/main/java/com/practicum/playlistmaker/media/ui/view/MediaFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.practicum.playlistmaker.media.ui.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.material.tabs.TabLayoutMediator
+import com.practicum.playlistmaker.R
+import com.practicum.playlistmaker.databinding.FragmentMediaBinding
+
+class MediaFragment: Fragment() {
+
+    private lateinit var binding: FragmentMediaBinding
+    private lateinit var tabLayoutMediator: TabLayoutMediator
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        super.onCreate(savedInstanceState)
+        binding = FragmentMediaBinding.inflate(layoutInflater)
+
+        binding.viewPager.adapter = MediaViewPagerAdapter(childFragmentManager, lifecycle)
+
+        tabLayoutMediator =
+            TabLayoutMediator(binding.tabLayout, binding.viewPager) { tab, position ->
+                tab.text = when (position) {
+                    0 -> resources.getString(R.string.favorites_tracks)
+                    1 -> resources.getString(R.string.playlists)
+                    else -> "tab"
```

Comment:
```suggestion
                    else -> resources.getString(R.string.playlists)
```
Лучше написать вот так и не хардкодить в коде строчки